cypress/integration/5.x.x/monex_spec.js

```diff
@@ -0,0 +1,32 @@
+describe('Monex', function() {
+  beforeEach('log in', function () {
+    cy.visit('/monex/')
+    cy.contains('Sign in as a dba user')
+    cy.get('.login-box-body')
+    cy.get(':nth-child(1) > .form-control').type('admin')
+    cy.get('.btn').click()
+  })
+  describe('index', function (){
+    it('should load start page', function() {
+      // Go to monex
+      cy.visit('/monex/index.html')
+      cy.get('.box-footer > .btn').click()
+    })
+
+  })
+  // Pending since index pages have console error
+  // see https://github.com/eXist-db/monex/issues/60
+  xdescribe('profiling', function () {
```

Comment:
Have you tried `describe.skip(...)` here?

Reply:
both work the same, for consistency i stuck with `x`

Reply:
Oh really, interesting!

Reply:
I still prefer .skip being more readable and more obvious. Also, it is not mentioned in the mocha docs front page and could therefore be considered discouraged style. Moreover, is there a alternative representation for `.only`. So, in the long run I would prefer to switch to .skip globally.